lca_activity_browser/app/ui/tabs/inventory.py

```diff
@@ -138,8 +138,13 @@ def __init__(self, parent):
         )
 
         inventory_layout = QtWidgets.QVBoxLayout()
+        # inventory_layout.addStretch(200)
```

Comment:
I think these are my old attempts to make the tables expand more. I think we'll eventually have to fix this via the size policy of the tables. I can't actually tell a difference now that this is commented out. From my part it's okay to delete these commented lines completely.

Reply:
actually, those were my attempts from yesterday... sizing in PyQt is horrible until one really understands it... and apparently, I haven't yet. I'll probably rewrite the contents of the inventory tab a bit... I am not sure the MaybeTables are the best way to do this... I find it rather complicated to understand

Reply:
Oh, yes. Should have been obvious from the line numbers, since these are new lines. I thought you just commented out some old things :smile: 
Layout is a real pain. Also can't say that i understand how it works. When we figure it out how to do it, the activity tab also really needs some work